SUI/SUI/ContentView.swift

```diff
@@ -5,14 +5,98 @@
+        static let profileTitle = "Профиль"
+        static let thousandNumber = 1000
+        static let fiveHundredNumber = 500
+        static let spacerEightyNumber = 80
+        static let spacerFourNumber = 400
+        static let toggleNumber = 150
+        static let zeroNumber = 0
+        static let cornerRadiusNumber = 10
+        static let transfersMoneyAlertButton = "Перевести"
+        static let openBankAccountText = "Открыть счёт"
+        static let responseNumber = 0.5
+        static let dampingFractionNumber = 0.7
+        static let blendDurationNumber = 0.3
+    }
+
+    @State var isError = false
+    @State var isOnToggle = false
+    @State var isAction = false
+    @State var isTransferAction = false
+    @State var isOpenBankAccount = false
+    @State var isTransferMoney = false
+    @State private var moneycount: String = Constants.emptyString
+    @State private var account: String = Constants.emptyString
+    @State var money = Constants.moneyCount
+    
     var body: some View {
         VStack {
-            Image(systemName: "globe")
-                .imageScale(.large)
-                .foregroundColor(.accentColor)
-            Text("Hello, world!")
-        }
-        .padding()
+            ZStack {
+                HStack {
+                    VStack {
+                        Text(Constants.walletString).frame(maxWidth: .infinity, alignment: .leading)
+                        Button {
+                            self.isAction = true
+                            money += Constants.thousandNumber
+                        } label: {
+                            Text(Constants.refillTheBalanceButtonTitle).frame(maxWidth: .infinity, alignment: .leading)
+                        }.frame(maxWidth: .infinity, alignment: .leading).actionSheet(isPresented: $isAction) {
+                            ActionSheet(title: Text(Constants.actionSheetRefillTheBalanceText))
+                        }
+                        
+                        Button {
+                            self.isTransferMoney = true
+                        } label: {
+                            Text(Constants.transferMoneyTitle).frame(maxWidth: .infinity, alignment: .leading)
+                        }.alert(isPresented: $isTransferMoney) {
+                            Alert(title: Text(Constants.transferMoneyTitle),
+                                  message: Text(Constants.transferMoneyMessage),
+                                  primaryButton: .cancel(),
+                                  secondaryButton: .default(Text(Constants.transferMoneyTitle), action: {
+                                guard money >= Constants.fiveHundredNumber else { return }
+                                money -= Constants.fiveHundredNumber
+                            }))
+                        }
+                        
+                        Spacer().frame(height: CGFloat(Constants.spacerEightyNumber))
```

Comment:
```suggestion
                        Spacer()
                             .frame(height: CGFloat(Constants.spacerEightyNumber))
```
Все модификаторы с новой